app/resources/subscribe_captcha.html.template

```diff
@@ -16,12 +16,23 @@
 {% block content %}
 
 <div>
-  <h2>{% blocktrans with person.primary_full_name as full_name %}Subscribe to updates about {{full_name}}{% endblocktrans %}</h2>
-  {% if subscribe_email %}
+
+  {% if context == 'create_person' %}
+    <h2>{% blocktrans with person_record_link_html|safe as full_name %}The record for {{full_name}} has been registered{% endblocktrans %}</h2>
+  {% else %}{% if context == 'add_note' %}
```

Comment:
I tried it and it didn't work. I believe we need to update the library to use it.